Source/Core/Core/NetPlayClient.cpp

```diff
@@ -632,6 +633,15 @@ void NetPlayClient::SendAsync(sf::Packet&& packet)
 // called from ---NETPLAY--- thread
 void NetPlayClient::ThreadFunc()
 {
+  std::unique_ptr<NetPlayQoSSession> qos_session;
```

Comment:
Nitpick, but the scope of `qos_session` seems a bit odd. I would include the declaration and the `AppendChat()` stuff in the if block.

Reply:
The `SConfig::GetInstance().bQoSEnabled` if block? If the declaration was inside that if block the QoS handle would be destroyed on Windows before the thread function starts running.

Reply:
Oh whoops, I didn't notice that, my bad.